Floofbot/Handlers/CommandHandler.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Reflection;
 using System.Threading.Tasks;
+using Discord;
```

Comment:
do we need to import the whole discord module?

Reply:
Yep. It's to ensure the MessageSource enum can be found.